### PR TITLE
Add instructions about how to get started with troubleshooting

### LINKED ISSUE #2679: Create a how-to guide for users seeking for troubleshooting
## Is your request for enhancement related to a problem? Please describe.
Some users might be confused about how to search troubleshooting tips on doc site or Zowe github repo. We can simply provide overall instructions to guide them finish troubleshooting. 

## Describe the solution you'd like
Adding a new page will do. The troubleshooting process could be:

1. Search the error message or code on doc site and see if there are existing solutions. 
2. If no solution available, users can search the error message or code in our GitHub repo to see if any issues or pull requests. [need a screenshot or .gif here]
3. If no issues or PRs available, users could reach out to open source community support channel or any other support service available.

### PR DESCRIPTION
**Pull request by samanthasusu**
Signed-off-by: samanthasusu <Wen.Ting.Su@ibm.com>
- Added a section to the overall troubleshooting page

### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [ ] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.
close #2679 
### Description (including links to related git issues)
Please describe your pull request.
Added some instructions about how to get started with troubleshooting. Feel free to leave your comments. Thanks!
:heart:Thank you!

### CODE REVIEW

**Review by Joe-Winchester — APPROVED** [review]
This is great !   I love the animated gif with the red boxes showing where to type.  Nice !

File: docs/troubleshoot/troubleshooting.md [hunk 1]
```diff
@@ -2,6 +2,29 @@
 
 To isolate and resolve Zowe&trade; problems, you can use the troubleshooting and support information.
 
+## How to start troubleshooting
+
+When you run into some issues and look for troubleshooting tips, the following steps may help you.
```
Comment (anaxceron) [inline docs/troubleshoot/troubleshooting.md:7]: How about "... some issues and **are looking** for troubleshooting ..."

File: docs/troubleshoot/troubleshooting.md [hunk 2]
```diff
@@ -2,6 +2,29 @@
...
+
+1. Search the error message or error code that you have in log on [Zowe doc site](https://docs.zowe.org/). If there are existing solutions, then you could follow them.
```
Comment (anaxceron) [inline docs/troubleshoot/troubleshooting.md:9]: When you mention "log," I think we need to specify what kind of log. Is it an _error_ log? A _deploy_ log? A _personal_ log? I think "log" alone can be confusing. Also, I'd explain how to search. "Search the error message or code listed in your error log by using the Search bar in the Zowe Docs site" (Remember, [Zowe Docs](https://docs.zowe.org/) in capitalized: See the main page.)

Comment (anaxceron) [reply docs/troubleshoot/troubleshooting.md:9]: Also suggesting: "If there is an existing solution, follow instructions to troubleshoot."

File: docs/troubleshoot/troubleshooting.md [hunk 3]
```diff
@@ -2,6 +2,29 @@
...
+
+2. If no solution is available or the existing solutions cannot apply to your problems, you could search the keywords, error messages, or error code in [Zowe GitHub repository](https://github.com/zowe). If the issues or pull requests are closed with solutions, you could follow them. If the issues are still open, you could leave your question or have a discussion in the thread to solve your issue.
```
Comment (anaxceron) [inline docs/troubleshoot/troubleshooting.md:11]: missing a "the": "... or error code in the Zowe GitHub repository." I'd also make "problems" singular.

Also suggesting: If you find a closed issue or pull request, try troubleshooting by using the information shared in the item's Conversation section. If an issue is still open, post a question or comment to prompt a discussion on your problem.

File: docs/troubleshoot/troubleshooting.md [hunk 4]
```diff
@@ -2,6 +2,29 @@
...
+
+![A gif to show how to search in github repo](../images/troubleshoot/how-to-troubleshooting.gif)
+
+3. If your problems are not solved, you could try the following options:
```
Comment (anaxceron) [inline docs/troubleshoot/troubleshooting.md:15]: Instead of "problems", I'd use the singular: "problem."

Comment (anaxceron) [reply docs/troubleshoot/troubleshooting.md:15]: Also suggesting: Be more direct in your instructions: "... not solved, try the following options:"

File: docs/troubleshoot/troubleshooting.md [hunk 5]
```diff
@@ -2,6 +2,29 @@
...
+
+* Create a new issue in the Zowe GitHub repository with detailed description of the problem that you encounter with.
```
Comment (anaxceron) [inline docs/troubleshoot/troubleshooting.md:17]: I'd provide a link to the repository
Correction: " .... with **a** detailed description ..."
Correction: No need for "with" or "that": " ... of the problem you have encountered."

File: docs/troubleshoot/troubleshooting.md [hunk 6]
```diff
@@ -2,6 +2,29 @@
...
+
+* Bring up your questions to the corresponding channels as shown below.
```
Comment (anaxceron) [inline docs/troubleshoot/troubleshooting.md:19]: Need a colon at the end of this sentence: " ... as shown below**:**"

**Review by anaxceron — COMMENTED** [review]
Hi @samanthasusu  -- left a few comments regarding grammar and word usage. Let me know if you have any questions.

**Comment by Joe-Winchester** [comment]
Sorry folks,  I merged this by mistake.   I had quite a few browser tabs open and was merging some other PRs and pushed this one by accident.  
Going to try put this back with:
- Restore the branch `add-a-how-to-for-troubleshooting`
- Created https://github.com/zowe/docs-site/pull/2686 to revert this merged PR and then merge 2686 back to master which was done with a force.
- For the last part @samanthasusu , can you please create a new PR for the branch `add-a-how-to-for-troubleshooting` as you're the original author ?  This will allow the review process to be re-started.   I can't see a way to get this PR's state to "unmerged" which is what I'd like, and reading github docs I think a new PR from the branch needs to be created fresh.

Sorry again for my mistake.  Cheers !

**Comment by samanthasusu** [comment]
> Sorry folks, I merged this by mistake. I had quite a few browser tabs open and was merging some other PRs and pushed this one by accident. Going to try put this back with:
> 
>     * Restore the branch `add-a-how-to-for-troubleshooting`
> 
>     * Created [Revert "Add instructions about how to get started with troubleshooting" #2686](https://github.com/zowe/docs-site/pull/2686) to revert this merged PR and then merge 2686 back to master which was done with a force.
> 
>     * For the last part @samanthasusu , can you please create a new PR for the branch `add-a-how-to-for-troubleshooting` as you're the original author ?  This will allow the review process to be re-started.   I can't see a way to get this PR's state to "unmerged" which is what I'd like, and reading github docs I think a new PR from the branch needs to be created fresh.
> 
> 
> Sorry again for my mistake. Cheers !

No worries Joe! Thank you for your review: )

**Pull request merged**